trimmer_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
+class TrimmerDecorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
```

Comment:
- [ ] According to the project specifications, your `TrimmerDecorator `class should inherit from a base decorator that you have not created yet. Please create the base decorator 👍